nomic/data_operations.py

```diff
@@ -700,31 +595,8 @@ def __repr__(self) -> str:
 
 class AtlasMapTags:
     """
-    Atlas Map Tag State
-
-    Tags are shared across all maps in your AtlasProject. You can manipulate tags by filtering over
-    the associated pandas dataframe
-
-    === "Accessing Tags Example"
-        ``` py
-        from nomic import AtlasProject
-
-        project = AtlasProject(name='My Project')
-        map = project.maps[0]
-        print(map.tags)
-        ```
-    === "Output"
-        ```
-              id_  oil  search_engines
-        0      0A    0               0
-        1      0g    0               0
-        2      0Q    0               0
-        3      0w    0               0
-        4      1A    1               0
-        ...   ...  ...             ...
-        9998  JZQ    0               0
-        9999  JZU    0               0
-        ```
+    Atlas Map Tag State. Tags are shared across all maps in your AtlasProject. You can manipulate tags by filtering over
```

Comment:
This is about to be not true for a while.

```suggestion
    Atlas Map Tag State. You can manipulate tags by filtering over
```